src/status_im/new_group/views/chat_group_settings.cljs

```diff
@@ -65,7 +67,8 @@
    type [:get :group-type]]
   (let [save-btn-enabled? (and (s/valid? ::v/name new-chat-name)
                                (not= new-chat-name chat-name))]
-    [view st/group-container
+    [(if ios? keyboard-avoiding-view view) (merge {:behavior :padding}
```

Comment:
It works fine, but Android already has its global solution in place (see my comment above) and the 2 don't play nice together